### ISSUE TITLE
AN181 Everspring / Mini Plug Switch with Metering (Gen5)  Amp-value cannot be polled

### ISSUE THREAD

**Issue by Nifrabar** [zwave-js/zwave-js-ui#3630]
zwave-js-ui: 9.9.0.09b107a
zwave-js: 12.4.4
home id: 3889492027
home hex: 0xe7d4f03b

controller:
Z‐Stick Gen5 USB Controller | ZW090

Device: 
AN181 Everspring / Mini Plug Switch with Metering (Gen5)
Problem several E-data must be polled to get it updated in domoticz.
It is not possible to poll Amp. value.

Is there a workaround?
-Bart

**Comment by Nifrabar** [comment]
yes I looked for amp. but it s not thereVerzonden vanaf Outlook voor Android

**Comment by robertsLando** [comment]
> yes I looked for amp. but it s not there

If you see it on UI it must be there for sure...

**Comment by Nifrabar** [comment]
it is on the UI and discovered with zwavejs2mqtt for domoticz. But as said earlier it is not updated so like to poll same as I had to do with other values of that device but no amp class to find. Here ar all possibilities.
![image](https://github.com/zwave-js/zwave-js-ui/assets/20126468/7ba56f5e-7df9-4f03-b09c-37d1beb84565)
![image](https://github.com/zwave-js/zwave-js-ui/assets/20126468/d26643e7-86ed-4ea0-8b34-ada7f56f66fb)
![image](https://github.com/zwave-js/zwave-js-ui/assets/20126468/076877df-b0dc-460f-8853-eba3822905d1)
![image](https://github.com/zwave-js/zwave-js-ui/assets/20126468/87fd3819-356d-4a01-a010-3d5724124414)
![image](https://github.com/zwave-js/zwave-js-ui/assets/20126468/e2e30264-a658-4287-97a8-bdaacc3b6be8)
![image](https://github.com/zwave-js/zwave-js-ui/assets/20126468/44e04c28-fc05-48fe-95c5-d86aeaabc38c)
Is there a workaround?

**Comment by robertsLando** [comment]
@Nifrabar Send me a screenshot of that value in control panel table please. It should be listed under Meter CC

**Comment by Nifrabar** [comment]
Domoticz represntation
![image](https://github.com/zwave-js/zwave-js-ui/assets/20126468/3ab9226c-c9f9-4011-8aa7-81655a2be7d4)
ZwaveJS-UI
![image](https://github.com/zwave-js/zwave-js-ui/assets/20126468/1a95d6ad-e507-439b-a1d2-4e4687440037)

**Comment by Nifrabar** [comment]
yesterday I sent s number of screencopies showing all. Amp for sure is not in the list.

**Comment by robertsLando** [comment]
@Nifrabar It's strange it's not there as all values you see in the UI should be listed there. Did you tried to restart your application and check again? I would need also a node debug export in order to debug this, just go to advanced actions on control panel and select debug export

**Comment by Nifrabar** [comment]
Mant these?

 

Van: Daniel Lando ***@***.***> 
Verzonden: woensdag 6 maart 2024 08:51
Aan: zwave-js/zwave-js-ui ***@***.***>
CC: Nifrabar ***@***.***>; Mention ***@***.***>
Onderwerp: Re: [zwave-js/zwave-js-ui] AN181 Everspring / Mini Plug Switch with Metering (Gen5) Amp-value cannot be polled (Issue #3630)

 

@Nifrabar <https://github.com/Nifrabar>  It's strange it's not there as all values you see in the UI should be listed there. Did you tried to restart your application and check again? I would need also a node debug export in order to debug this, just go to advanced actions on control panel and select debug export

**Comment by Nifrabar** [comment]
Well, maybe they where zipped.
[zwavejs_2024-02-29.log](https://github.com/zwave-js/zwave-js-ui/files/14511953/zwavejs_2024-02-29.log)

**Comment by Nifrabar** [comment]
Maybe because these were zipped.

[zwavejs_logrotate.json](https://github.com/zwave-js/zwave-js-ui/files/14511972/zwavejs_logrotate.json)
[zwavejs_current.log](https://github.com/zwave-js/zwave-js-ui/files/14511974/zwavejs_current.log)
[zwavejs_2024-02-29.log](https://github.com/zwave-js/zwave-js-ui/files/14511979/zwavejs_2024-02-29.log)

**Comment by robertsLando** [comment]
Nope those are logs, I need you to go to Advanced > Dump Export
![2024-03-06_17-46](https://github.com/zwave-js/zwave-js-ui/assets/11502495/8fa9a30e-d069-4f5a-925f-24ed358f067c)

**Comment by Nifrabar** [comment]
Van: Daniel Lando ***@***.***> 
Verzonden: woensdag 6 maart 2024 17:47
Aan: zwave-js/zwave-js-ui ***@***.***>
CC: Nifrabar ***@***.***>; Mention ***@***.***>
Onderwerp: Re: [zwave-js/zwave-js-ui] AN181 Everspring / Mini Plug Switch with Metering (Gen5) Amp-value cannot be polled (Issue #3630)

 

Nope those are logs, I need you to go to Advanced > Dump Export
2024-03-06_17-46.png (view on web) <https://github.com/zwave-js/zwave-js-ui/assets/11502495/8fa9a30e-d069-4f5a-925f-24ed358f067c>

**Comment by Nifrabar** [comment]
[nodes_dump (4).json](https://github.com/zwave-js/zwave-js-ui/files/14525792/nodes_dump.4.json)

**Comment by robertsLando** [comment]
@Nifrabar Ok the issue is you have multiple devices with same manufacturer but different `productId`.

In this case the only device with power consumption is the one with device id: `96-6-4`  `AN181`
The ones without it are: `96-2-4` `AN158`

So the issue is that when you select the divice in the first dropdown you have to select the `96-6-4` then you will see the ampere value in the value dropdown
